rhel/SPECS/prometheus.spec

```diff
@@ -7,24 +7,19 @@
 %doc %{*} \
 %endif
 
-%global provider        github
-%global provider_tld    com
-%global project         prometheus
-%global repo            prometheus
-%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
-%global import_path     %{provider_prefix}
-%global commit          b90be6f32a33c03163d700e1452b54454ddce0ec
+%global import_path     github.com/prometheus/prometheus
```

Comment:
where is it used?

Reply:
Yes, it's not used anymore, `import_path` removed